hibernate-core/src/main/java/org/hibernate/action/internal/EntityInsertAction.java

```diff
@@ -263,13 +250,12 @@ protected boolean cacheAfterInsert(EntityDataAccess cache, Object ck) {
 
 	@Override
 	protected boolean hasPostCommitEventListeners() {
-		final EventListenerGroup<PostInsertEventListener> group = listenerGroup( EventType.POST_COMMIT_INSERT );
+		final EventListenerGroup<PostInsertEventListener> group = getFastSessionServices().eventListenerGroup_POST_COMMIT_INSERT;
 		for ( PostInsertEventListener listener : group.listeners() ) {
```

Comment:
I think `getFastSessionServices().eventListenerGroup_POST_COMMIT_INSERT` can be `null`

Reply:
sorry looking at the code and it seems not